packages/vector_graphics_compiler/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.1.16
+
+* Fixes an issue when parsing stroke-width with an invalid value
```

Comment:
Maybe something like "Defaults stroke-width to 1 when an invalid value is parsed instead of throwing an exception"

Reply:
i will adjust this